### PR TITLE
Vsts2092

### PR DESCRIPTION
**Pull request by VaclavK**

### CODE REVIEW

File: src/Eshopworld.Web/Configuration/CoreConfiguration.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
+﻿using Microsoft.Extensions.Configuration;
+
+namespace Eshopworld.Web.Configuration
+{
+    /// <summary>
+    /// Core configuration builder implementation with specific loading sequence built in
+    /// </summary>
+    public static class CoreConfiguration
```
Comment (djfr) [inline src/Eshopworld.Web/Configuration/CoreConfiguration.cs]: CoreConfiguration needs to be taken out of Eshopworld.Web. It is used in projects like webjobs that can't take the web dependencies.

File: src/Eshopworld.Web/Configuration/SectionKeyVaultManager.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+{
+    /// <summary>
+    /// this class plugs in the Configuration Extensions API and loads keys into dedicated naming scope
+    /// 
+    /// naming structure of secrets as expected
+    /// 
+    /// appName-sectionName-secretName (e.g .fraud_api-database-connString)
+    /// 
+    /// </summary>
+    public class SectionKeyVaultManager : IKeyVaultSecretManager
```
Comment (djfr) [inline src/Eshopworld.Web/Configuration/SectionKeyVaultManager.cs]: This needs to be taken out also. Same as CoreConfiguration.

File: src/Eshopworld.Web/Eshopworld.Web.csproj [hunk 3]
```diff
@@ -0,0 +1,33 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <IsPackable>True</IsPackable>
+    <PackageVersion>1.0.0</PackageVersion>
```
Comment (djfr) [inline src/Eshopworld.Web/Eshopworld.Web.csproj]: Don't use PackageVersion, but Version instead, it will version the DLL and the package.

Comment (VaclavK) [reply src/Eshopworld.Web/Eshopworld.Web.csproj]: oh yeah! completely slipped my mind - I remember you mentioning that

**Pull request merged**